src/doc/src/reference/manifest.md

```diff
@@ -288,6 +288,8 @@ along with the defaults for each profile.
 [profile.dev]
 opt-level = 0      # controls the `--opt-level` the compiler builds with.
                    # 0-1 is good for debugging. 2 is well-optimized. Max is 3.
+                   # 's' will only run optimizations that do not increase
+                   # binary size, 'z' will optimize for size aggressively.
```

Comment:
Aren't `-s` and `-z` nightly only?

Reply:
Hm, I never used any ````-Z ..```` flag to enable these, but lemme check...

Reply:
You're right!

Reply:
Updated, thanks!